pkgs/applications/misc/bottles/fhsenv.nix

```diff
@@ -1,7 +1,6 @@
 { buildFHSEnv
 , symlinkJoin
 , bottles-unwrapped
-, gst_all_1
```

Comment:
It was used in `multiPkgs`. You can't see it in the diff, you have to view the entire file.

After removing it, the various gstreamer plugins now use the correct `pkgs` (the one from `multiPkgs`, instead of the one from `callPackage`).

Reply:
Ohhh dang; the `with;` statement strikes again.

Since this is so unintuitive, it'd be better to include the info how this works in the commit message and/or PR description in order to not confuse reviewers.